components/serializers.py

```diff
@@ -72,8 +74,38 @@ class Meta:
             "catalog_data",
             "component_data",
             "project_data",
+            "component_json",
         )
 
+    def partial_update(self, instance, validated_data):
```

Comment:
Ignore for now, going to try to utilize this as a patch call instead of a new endpoint with a put...